cmd/influxd/launcher/launcher.go

```diff
@@ -269,29 +281,52 @@ func (m *Launcher) run(ctx context.Context, opts *InfluxdOpts) (err error) {
 
 	var flushers flushers
 	switch opts.StoreType {
-	case BoltStore:
-		store := bolt.NewKVStore(m.log.With(zap.String("service", "kvstore-bolt")), opts.BoltPath)
-		store.WithDB(m.boltClient.DB())
-		m.kvStore = store
+	case DiskStore, BoltStore:
```

Comment:
Should we log a warning if `BoltStore` is used?

Reply:
Seems like a good idea, I made it log a warning if `BoltStore` is specified using the normal logger. Let me know if there's a better way to make this more visible - I wasn't able to easily find other examples, but certainly could have missed something. Probably a good thing to put in the release notes as well.